autoload/bufferline.vim

```diff
@@ -0,0 +1,51 @@
+function! bufferline#enable()
```

Comment:
All of these functions are still there, but just point to the Lua `bufferline` module now. If I got rid of them it might break something someone's made :grimacing: